rtp_cluster.py

```diff
@@ -76,7 +76,7 @@ def reopen(signum, logfile):
     global_config = {}
 
     try:
-        opts, args = getopt.getopt(sys.argv[1:], 'fP:c:L:s:o:dD')
+        opts, args = getopt.getopt(sys.argv[1:], 'fP:c:L:s:o:dDb:')
```

Comment:
@NormB Actually the colon signifies options that "require" an argument/value.  So if you specify the '-b' option, you have to pass in the appropriate value with it.